daniellamarr/fast-food-fast
# #160381337 Setup coveralls test coverage

#### What does this PR do?
- This PR integrates coveralls and code climate badges to the app
#### Description of Task to be completed?
- Integrate badges to the README
#### How should this be manually tested?
- To view coveralls report, pull this branch **ch-setup-coveralls-test-coverage-160381337** off this repository
- Run npm install on your local machine to install all dependencies
- Run npm test to see coverall report
- Code climate report can only be seen online therefore see README for badges
#### Any background context you want to provide?
- This app is a **node app** and would require **nodejs** and **npm** installation on your local machine
#### What are the relevant pivotal tracker stories?
[160381337](https://www.pivotaltracker.com/story/show/160381337)
#### Screenshots (if appropriate)
None
#### Questions:
None
